README.md

```diff
@@ -2,32 +2,29 @@
-- [Navigating Compliance](./docs/navigating-compliance.md)
-- [Navigating Intended](./docs/navigating-intended.md)
-- [Navigating SoTAgg](./docs/navigating-sot-agg.md)
-- [FAQ](./docs/FAQ.md)
-
 # Overview
 
+You may see the [Navigating Overview](./docs/navigating-golden.md) documentation for an overview of navigating through the different areas of this plugin. You may also see the [FAQ](./docs/FAQ.md) for commonly asked questions.
+
 The golden configuration plugin performs four primary actions, each of which can be toggled on with a respective `enable_*` setting, covered in detail 
 later in the readme. 
 
 * Configuration Backup - Is a Nornir process to connect to devices, optionally parse out lines/secrets, backup the configuration, and save to a Git repository.
+    * see [Navigating Backup](./docs/navigating-backup.md) for more information
 * Configuration Intended - Is a Nornir process to generate configuration based on a Git repo of Jinja files and a Git repo to store the intended configuration.
+    * see [Navigating Intended](./docs/navigating-intended.md) for more information
 * Source of Truth Aggregation - Is a GraphQL query per device with that creates a data structure used in the generation of configuration.
-* Configuration Compliance - Is a Nornir process to run comparison of the actual (via backups) and intended (via Jinja file creation) cli configurations.
+    * see [Navigating SoTAgg](./docs/navigating-sot-agg.md) for more information
+* Configuration Compliance - Is a Nornir process to run comparison of the actual (via backups) and intended (via Jinja file creation) CLI configurations.
+    * see [Navigating Compliance](./docs/navigating-compliance.md) for more information
 
 The operator's of their own Nautobot instance are welcome to use any combination of these features. Though the appearance may seem like they are tightly 
-coupled, they are not actually. As an example, one can obtain backup configurations from their current RANCID/Oxidized process and simply provide a Git Repo
-of the location of the backup configurations and the compliance process would work the same way. Another user may only want to generate configurations.
+coupled, this isn't actually the case. For example, one can obtain backup configurations from their current RANCID/Oxidized process and simply provide a Git Repo
+of the location of the backup configurations, and the compliance process would work the same way. Also, another user may only want to generate configurations, but not want to use other features, which is perfectly fine to do so.
 
 ## Screenshots
 
-There are many features and capabilities the plugin provides, the following is intended to provide a quick visual overview of some of those features.
+The plugin provides features and capabilities to integrate more workflows closely into the Nautobot ecosystem. The following screenshots are intended to provide a quick visual overview of some of these features.
```

Comment:
```suggestion
There are many features and capabilities the plugin provides into the Nautobot ecosystem. The following screenshots are intended to provide a quick visual overview of some of these features.
```

Reply:
> The plugin provides features and capabilities to integrate into the Nautobot ecosystem. The following screenshots are intended to provide a quick visual overview of some of these features.

if this sentence can be used instead it should make more sense. cheers